crates/rolldown_binding/src/options/binding_input_options/binding_experimental_options.rs

```diff
@@ -1,4 +1,52 @@
-#[napi_derive::napi(object)]
+use napi::{
+  Error, Status,
+  bindgen_prelude::{FromNapiValue, sys},
+  check_status,
+};
+
+#[repr(transparent)]
+#[derive(Debug)]
+pub struct SourcemapHires(string_wizard::Hires);
+
+impl FromNapiValue for SourcemapHires {
```

Comment:
We don't really prefer pattern of implementing `FromNapiValue` manually.

We probably would use `Either<Boolean, String>` in the binding options and convert it to the real hire option in

https://github.com/rolldown/rolldown/blob/d227c81088b30e2357af602881716f3e3a189b3e/crates/rolldown_binding/src/utils/normalize_binding_options.rs#L304